src/PatientChecking/PatientChecking/Views/Dashboard/Home.cshtml

```diff
@@ -2,4 +2,67 @@
     ViewData["Title"] = "Dashboard";
     ViewData["Page"] = "Dashboard";
 }
-<p>This is dashboard</p>
+@section css {
+    <link href="@Url.Content("~/css/Dashboard.css")" rel="stylesheet" type="text/css" />
+}
+<script>
+    (function () {
+        $('#loading-div-background').show();
+        
+        $.ajax({
+            type: 'GET',
+            url: 'https://localhost:5001/Dashboard/getDashBoard',
+            success: function (data) {
+                $(".total-patient").text(data.numOfPatients);
+                $(".patient-inmonth").text(data.numOfPatientsInMonth + " new paitents/this month");
+                $(".total-appointment").text(data.numOfAppointments);
+                $(".appointment-inmonth").text(data.numOfAppointmentsInMonth + " new appointments/this month");
+                $(".appointment-intoday").text(data.numOfAppointmentsInToday + " new appointments/today");
+                $('#loading-div-background').hide();
+                $(".grid-wide").show();
+            }
+        })
+    })();
+</script>
+<div id="loading-div-background" style="position: fixed; top: 50%; left: 60%;">
```

Comment:
please don't include in-line styling, instead try to use bootstrap class.

Reply:
I customized Loading tag, so we can use to other places